src/main/blackbox/blackbox_io.h

```diff
@@ -72,8 +72,10 @@ bool blackboxDeviceFlushForce(void);
 bool blackboxDeviceOpen(void);
 void blackboxDeviceClose(void);
 
+#ifdef USE_FLASHFS
 void blackboxEraseAll(void);
 bool isBlackboxErased(void);
+#endif
```

Comment:
I generally prefer not to include the build flags in the headers. It makes them less easy to read and serves no real advantage.